stylus/components/chip.styl

```diff
@@ -1,26 +1,58 @@
 @require 'settings/palette'
 @require 'tools/mixins'
 
-$chip-height = rem(40)
-$chip-font-size = rem(16)
+$chip-height-small = rem(32)
+$chip-font-size-small = rem(14)
+
+$chip-height-normal = rem(40)
+$chip-font-size-normal = rem(16)
 
 $chip
-    border-radius ($chip-height / 2)
-    height $chip-height
-    background var(--paleGrey)
-    padding (($chip-height - $chip-font-size) / 2)
     box-sizing border-box
     line-height 1
     display inline-flex
     align-items center
     margin-right rem(4)
     margin-bottom rem(4)
+    border-width 0
 
 $round-chip
-    width $chip-height
+    width $chip-height-normal
     text-align center
     justify-content center
 
+$small-size-chip
+    height $chip-height-small
+    padding (($chip-height-small - $chip-font-size-small) / 2)
+    border-radius ($chip-height-small / 2)
+    font-size $chip-font-size-small
+
+$normal-size-chip
+    height $chip-height-normal
+    padding (($chip-height-normal - $chip-font-size-normal) / 2)
+    border-radius ($chip-height-normal / 2)
+
+$outlined-variant-chip
+    border 1px solid
+    // important is here because this variant needs to have a transparent
+    // background, no matter with which theme it is used
+    background-color transparent !important // @stylint ignore
```

Comment:
I don't like the important here but I do not see an easy win.

Reply:
It's either that or increasing the specifity weight in `react/Chip/styles.styl` like you did L30 ( which I tend to prefer) or create a new theme… :/